guides/common/modules/ref_pxe-loaders.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="pxe-loaders"]
+= PXE loaders
+
+In {Project}, a PXE loader defines the `filename` DHCPv4 option to use during provisioning.
+{Project} deploys the required files to TFTP {SmartProxy}.
+
+.Limitations
+* {Project} can deploy the required booting files only for the `x86_64` architecture.
+ifdef::satellite[]
+* {Team} does not provide support for iPXE workflows.
+endif::[]
+
+.PXE loaders and DHCPv4 filenames
+|===
+| PXE loader option | `filename` value | Use for
+
+| *None*
+| No filename passed
+| Non-PXE booting, for example HTTP booting with iPXE or image-based provisioning
+
+| *PXELinux BIOS*
+| `pxelinux.0`
+| BIOS machines with PXE booting
+
+| *PXELinux UEFI*
+| `pxelinux.efi`
+| UEFI machines with PXE booting
+
+| *Grub2 BIOS*
+| `grub2/grub__arch__.0`
+| BIOS machines with PXE booting
+
+| *Grub2 UEFI*
+| `__subdir__/grub2/grub__arch__.efi`
+| UEFI machines with PXE booting
+
+| *Grub2 UEFI SecureBoot*
+| `__subdir__/grub2/shim__arch__.efi`
+| SecureBoot-enabled UEFI machines with PXE booting
+
+| *Grub2 UEFI HTTP*
+| `http://__httpboot.example.com__/httpboot/__subdir__/grub2/grub__arch__.efi`
+| UEFI machines with HTTP booting
+
+| *Grub2 UEFI HTTPS*
+| `https://__httpboot.example.com__/httpboot/__subdir__/grub2/grub__arch__.efi`
```

Comment:
Similar to above, this is served from the Smart Proxy but this time on the HTTPS interface. That listens on `{smartprox_port}` by default.